### ISSUE TITLE
Change screen summary interaction hint to reference "Photon Rate" slider

### ISSUE THREAD

**Issue by terracoda** [phetsims/molecules-and-light#240]
In issue #234, we changed the label on the slider in the Observation Window.

Please make a similar change to the initial interaction hint at the end of the Screen Summary.

Specifically, change:
 - Look for **light source** slider to play.

To:
- Look for **photon rate** slider to play.

Other adjustments to the screen summary can take place after interviews next week.

**Comment by jessegreenberg** [comment]
Great, done in the above commit.

**Comment by terracoda** [comment]
@jessegreenberg , I should have mentioned that this change was dependent on implementing changes in https://github.com/phetsims/molecules-and-light/issues/234#issuecomment-564070461

Can I assume those changes are coming, too.

**Comment by terracoda** [comment]
If so, safe to close.

**Comment by jessegreenberg** [comment]
Definitely, ill make that change in that issue.
